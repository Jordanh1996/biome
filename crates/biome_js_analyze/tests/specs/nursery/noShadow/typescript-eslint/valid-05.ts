/* should not generate diagnostics */
const test = 1;
type Fn = (test: string) => typeof test;
