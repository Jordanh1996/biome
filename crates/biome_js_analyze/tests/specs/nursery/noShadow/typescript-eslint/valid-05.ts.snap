---
source: crates/biome_js_analyze/tests/spec_tests.rs
expression: valid-05.ts
---
# Input
```ts
/* should not generate diagnostics */
const test = 1;
type Fn = (test: string) => typeof test;

```
